### ISSUE TITLE
[deeplab][feature request] Checkpoint for MobileNet v2 (dw multiplier=0.75)

### ISSUE THREAD

**Issue by kushagraagrawal** [tensorflow/models#4409]
### System information
- **What is the top-level directory of the model you are using**:deeplab
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 14.04
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: 1.7
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**: 9.0
- **GPU model and memory**:
- **Exact command to reproduce**:

### Describe the problem
I wish to use the deeplab decoder with MobileNet v2 with depth multiplier=0.75. Can anyone provide a checkpoint for that? Or how do I train it on my own with the available checkpoints?

**Comment by kushagraagrawal** [comment]
@aquariusjay could you help regarding this issue?

**Comment by aquariusjay** [comment]
Hi kushagraagrawal ,

We currently do not have any plan to release a MobileNetv2 (dm=0.75) pretrained checkpoint.
But, you should be able to train one by yourself with the ImageNet pretrained [checkpoint](https://github.com/tensorflow/models/tree/master/research/slim/nets/mobilenet), since MobileNetv2 model variants do not require too much GPU memory. The codebase should be able to support different depth multiplier (see [here](https://github.com/tensorflow/models/blob/master/research/deeplab/core/feature_extractor.py#L33)).

**Comment by kushagraagrawal** [comment]
Hi aquariusjay, I tried training with dm=0.75, using the following command

`NUM_ITERATIONS=5000
python "${WORK_DIR}"/train.py \
  --logtostderr \
  --train_split="trainval" \
  --model_variant="mobilenet_v2" \
  --output_stride=16 \
  --depth_multiplier=0.75 \
  --train_crop_size=321 \
  --train_crop_size=321 \
  --train_batch_size=32 \
  --training_number_of_steps="${NUM_ITERATIONS}" \
  --tf_initial_checkpoint="${INIT_FOLDER}/${CKPT_NAME}/mobilenet_v2_0.75_224.ckpt" \
  --fine_tune_batch_norm=true \
  --train_logdir="${TRAIN_LOGDIR}" \
  --dataset_dir="${PASCAL_DATASET}" \
`

However, the loss seems to stagnate at 1.x with mIoU being 3.5%. Any suggestions?

**Comment by KleinXin** [comment]
@kushagraagrawal  Have you solved this problem?

I am also trying to use the imagenet, pascal or coco pretrained model to do the finetuning.

When I set --depth_multiplier=0.75 as you suggested above, it seems the model is not initialised by the imagenet pretrained model because the following words are shown.

"W0726 09:55:51.190753 140104241526528 variables.py:676] Checkpoint is missing variable [image_pooling/weights]
W0726 09:55:51.190967 140104241526528 variables.py:676] Checkpoint is missing variable [image_pooling/BatchNorm/gamma]
W0726 09:55:51.191039 140104241526528 variables.py:676] Checkpoint is missing variable [image_pooling/BatchNorm/beta]
W0726 09:55:51.191099 140104241526528 variables.py:676] Checkpoint is missing variable [image_pooling/BatchNorm/moving_mean]
W0726 09:55:51.191173 140104241526528 variables.py:676] Checkpoint is missing variable [image_pooling/BatchNorm/moving_variance]
W0726 09:55:51.191290 140104241526528 variables.py:676] Checkpoint is missing variable [aspp0/weights]
W0726 09:55:51.191353 140104241526528 variables.py:676] Checkpoint is missing variable [aspp0/BatchNorm/gamma]
W0726 09:55:51.191409 140104241526528 variables.py:676] Checkpoint is missing variable [aspp0/BatchNorm/beta]
W0726 09:55:51.191464 140104241526528 variables.py:676] Checkpoint is missing variable [aspp0/BatchNorm/moving_mean]
W0726 09:55:51.191519 140104241526528 variables.py:676] Checkpoint is missing variable [aspp0/BatchNorm/moving_variance]
W0726 09:55:51.191573 140104241526528 variables.py:676] Checkpoint is missing variable [concat_projection/weights]
W0726 09:55:51.191627 140104241526528 variables.py:676] Checkpoint is missing variable [concat_projection/BatchNorm/gamma]
W0726 09:55:51.191681 140104241526528 variables.py:676] Checkpoint is missing variable [concat_projection/BatchNorm/beta]
W0726 09:55:51.191735 140104241526528 variables.py:676] Checkpoint is missing variable [concat_projection/BatchNorm/moving_mean]
W0726 09:55:51.191789 140104241526528 variables.py:676] Checkpoint is missing variable [concat_projection/BatchNorm/moving_variance]
W0726 09:55:51.191843 140104241526528 variables.py:676] Checkpoint is missing variable [logits/semantic/weights]
W0726 09:55:51.191897 140104241526528 variables.py:676] Checkpoint is missing variable [logits/semantic/biases]
"

I think that is why the mIoU is very low. In fact, the model is trained from scratch instead of using transfer learning. The mIoU I get is 0.0147772571

@aquariusjay suggested to rewrite the codes in deeplab since it is not compatible with the mobilenet v2 imagenet pretrained model as it is said in this post

https://github.com/tensorflow/models/issues/5507

Hope we can find the solution

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
